spec/lib/percy/capybara/loaders/base_loader_spec.rb

```diff
@@ -14,6 +14,18 @@
       expect(resource.content).to include('Hello World!')
       expect(resource.sha).to eq(Digest::SHA256.hexdigest(resource.content))
     end
+    it 'includes an iframe resource' do
```

Comment:
Hmm, yes that is a bit of setup. My only concern here is that we haven't verified that the iframe logic is safe to run against RackTest (ie. when `js: false`, the default)—if iframe logic was behind a flag I'd be less concerned, but since we're enabling it by default it's more important to make sure it works in all environments.

Would you mind taking a stab at adding the test for this? Don't worry about adding a `#root_html_resource` test, just the iframe one is sufficient.

Quick stab:
```ruby
IFRAME_PATH = File.expand_path('../../client/testdata/test-iframe.html', __FILE__)

class RackAppWithIframe
  def self.call(env)
    [200, {}, File.read(IFRAME_PATH)]
  end
end

#...

  context 'Rack::Test', type: :feature do
    before(:each) { Capybara.app = RackAppWithIframe }
    after(:each) { Capybara.app = nil }
    # ... duplicate the `it` test here
  end
```

However! Note that `Rack::Test` is super weird and does not run a browser underneath, so there may not be any iframe source at all. I just want to make sure it runs without errors, not that it works correctly (I'll be surprised if it does). We will probably have to say that `js: true` is required for iframe support.